airbyte-db/lib/src/main/java/io/airbyte/db/instance/jobs/migrations/V0_35_5_001__Add_failureSummary_col_to_Attempts.java

```diff
@@ -21,6 +22,11 @@ public void migrate(final Context context) throws Exception {
     LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
 
     final DSLContext ctx = DSL.using(context.getConnection());
+    migrate(ctx);
+  }
+
+  @VisibleForTesting
+  public static void migrate(final DSLContext ctx) {
```

Comment:
What is the purpose of this change?

Reply:
Ah just saw that this is used in the test below, disregard

Reply:
I think this is a better pattern for making migrations callable from unit tests (I took it from https://github.com/airbytehq/airbyte/blob/master/airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_32_8_001__AirbyteConfigDatabaseDenormalization.java#L62-L75). 

That said, it does seem strange to modify migrations that we've already run, so maybe I'll revert it here